File: CSharpShared/CSharpShared/RabbitMqMessages/UserCreatedMessage.cs
namespace CSharpShared.RabbitMqMessages;

public class UserCreatedMessage : RabbitMqMessage
{
    public long UserId { get; set; }
    public const string ConstType = "UserCreated";
    
    public UserCreatedMessage(long userId) : base(ConstType)
    {
        UserId = userId;
    }
}

File: notification_service/src/main/java/via/group1/notification_service/rabbitmq/messages/UserCreatedMessage.java
package via.group1.notification_service.rabbitmq.messages;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class UserCreatedMessage extends RabbitMqMessage{
    long userId;
    public UserCreatedMessage(long userId) {
        super("UserCreated");
        this.userId = userId;
    }

    public UserCreatedMessage() {
        super("UserCreated");
    }
}
